test_requirements.txt

```diff
@@ -1 +1,2 @@
-pytest==7.0.1
\ No newline at end of file
+pytest>=7.0.1
+pytest-mock
```

Comment:
Can you pin the version of pytest-mock to the one you are using?

Reply:
I used latest whatever it is, I'm always using a fresh virtual environnement